src/golang/cmd/server/server/aqueduct_server.go

```diff
@@ -330,3 +335,16 @@ func (s *AqServer) Restart() {
 	s.RequestMutex.Unlock()
 	s.UnderMaintenance.Store(false)
 }
+
+// Generates the full server address for display purpose.
+// If `ExternalIP` is configured, it returns `[ExternalIP]:[Port]`.
+// Otherwise it returns `<IP_ADDRESS>:[PORT]` expecting the user to replace
+// `<IP_ADDRESS>` string while using this information.
```

Comment:
"while using this information" what does this information refer to? Maybe just say "expecting the user to manually replace `<IP_ADDRESS>` with the server's IP"?